README.md

```diff
@@ -1,39 +1,50 @@
-Raspberry Pi Board Support Package for Windows 10 IoT Core
+Raspberry Pi Board Support Package for Windows 10
 ==============
 
-## Welcome to the Raspberry Pi Board Support Package (BSP) for Windows 10 IoT Core
+## Welcome to the Raspberry Pi Board Support Package (BSP) for Windows 10
 
-This repository contains BSP components for the Raspberry Pi 2, 3, and Compute Module. This BSP repository is under community support; it is functional with the Fall 2018 release of Windows 10 IoT Core but is not actively maintained by Microsoft. BSP elements included in this repository may contain features that are not available with Windows 10 IoT Core releases.
+This repository contains BSP components for the Raspberry Pi 2, 3 and Compute Modules running 32-bit Windows 10 IoT Core (version 1809).
+It also contains BSP components for the Raspberry Pi 2 (v1.2), 3, 4 and corresponding Compute Modules running 64-bit Windows 10.
 
-For more information about Windows 10 IoT Core, see online documentation [here](http://windowsondevices.com)
+This BSP repository is under community support; it is not actively maintained by Microsoft. 
+For Raspberry Pi 4 onwards, it's designed for use with Windows 10 version 2004 and later, because of use of newer driver frameworks which are not available on earlier releases.
 
 This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
 
-## Firmware binaries
+## 32-bit firmware
 
 Sample binaries of the firmware is included in [RPi.BootFirmware](bspfiles/Packages/RPi.BootFirmware) to enable quick prototyping. The sources for these binaries are listed below.
 
 1. Firmware binaries : [RaspberryPi/Firmware](https://github.com/raspberrypi/firmware)
 2. UEFI Sources : [RPi/UEFI](https://github.com/ms-iot/RPi-UEFI)
 
-### UEFI Customisations
+## 64-bit firmware
+
+For the Raspberry Pi 4, firmware from the [Pi Firmware Task Force](https://rpi4-uefi.dev) is used, which provides UEFI and ACPI support.
```

Comment:
Recommend also adding a link to the RPi4 UEFI releases page - https://github.com/pftf/RPi4/releases